aggregator/src/main/resources/application.properties

```diff
@@ -8,7 +8,7 @@ quarkus.http.port=8086
 quarkus.datasource.db-kind=postgresql
 quarkus.datasource.username=postgres
 quarkus.datasource.password=postgres
-quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications_backend
+quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications
```

Comment:
Shouldn't this remain as before ?

Reply:
It should match https://github.com/RedHatInsights/notifications-backend/blob/master/backend/src/main/resources/application.properties#L44 in my understanding, so I think that change is good.

Reply:
I would say it should be the same as in backend, but with non-reactive: 
quarkus.datasource.reactive.url=postgresql://127.0.0.1:5432/notifications

Reply:
Yes, sorry, it just needs to match.
I got confused with the name [we use in OpenShift environments](https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/notifications/notifications-aggregator.yml#L69)
But as long as they match in their respective environments, we're good